eatsmart/locations/wake/management/commands/import_wake.py

```diff
@@ -154,3 +174,39 @@ def save_violations(self, violations):
                 print('New record')
             violation_obj.__dict__.update(**attributes)
             violation_obj.save()
+
+    def inspection_risk_factors(self):
+        factors = (('hold_temp', 1), ('cook_temp', 2), ('contamination', 3),
+                   ('hygeine', 4), ('source', 5))
+        inspections = Inspection.objects.all()
+        for inspection in inspections:
+            violations = Violation.objects.filter(inspection_id=inspection.id)
```

Comment:
You should be able to do this here:
violations = inspection.violation_set.all()